src/styles/blocks/card.scss

```
@@ -0,0 +1,85 @@
+.card {
+  display: flex;
+  max-width: 200px;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors

```suggestion
  border: 1px solid $c-color;
```